source/common/common/logger.h

```diff
@@ -563,22 +563,26 @@ class ExtractedMessage : public spdlog::custom_flag_formatter {
 #define ENVOY_TAGGED_CONN_LOG_TO_LOGGER(LOGGER, LEVEL, TAGS, CONNECTION, FORMAT, ...)              \
   do {                                                                                             \
     if (ENVOY_LOG_COMP_LEVEL(LOGGER, LEVEL)) {                                                     \
-      TAGS.emplace("ConnectionId", std::to_string((CONNECTION).id()));                             \
-      ENVOY_LOG_TO_LOGGER(LOGGER, LEVEL,                                                           \
-                          fmt::runtime(::Envoy::Logger::Utility::serializeLogTags(TAGS) + FORMAT), \
-                          ##__VA_ARGS__);                                                          \
+      std::map<std::string, std::string> log_tags = TAGS;                                          \
+      log_tags.emplace("ConnectionId", std::to_string((CONNECTION).id()));                         \
```

Comment:
The implementation contradicts the PR description `the printed log will only include tags from the inline map, and will not include the connection/stream ID tags`. Which one is correct?

Reply:
In the description I meant that currently the bug is that if you were to log with:
``ENVOY_TAGGED_CONN_LOG(info, (std::map<std::string, std::string>{{"key_inline", "val"}}), connection_, "fake message")``
The output would be: ``[Tags: "key_inline":"val"] fake message``, where it should actually be: ``[Tags: "key_inline":"val","ConnectionId":"10"] fake message``.
The PR fixes this so logs will be printed correctly